avocado/utils/pci.py

```diff
@@ -52,7 +52,7 @@ def get_pci_addresses():
     addresses = []
     cmd = "lspci -D"
     for line in process.system_output(cmd).splitlines():
-        if "PCI bridge" not in line and "System peripheral" not in line:
+        if '06' not in get_pci_prop(line.split()[0], 'Class'):
```

Comment:
This will match with `06` in sub-classes as well (i.e. `Class: 0206`). It should be:
```python
    pci_class = get_pci_prop(line.split()[0], 'Class')
    if not pci_class.startswith('06'):
        ...
```